phpstan.neon

```diff
@@ -3,6 +3,18 @@ parameters:
     treatPhpDocTypesAsCertain: false
     ignoreErrors:
         - identifier: argument.type
+          paths:
+            - src/core/etl/tests/*
+            - src/bridge/filesystem/azure/tests/*
+            - src/bridge/filesystem/async-aws/tests/*
+            - src/adapter/etl-adapter-xml/tests/*
+            - src/adapter/etl-adapter-text/tests/*
+            - src/adapter/etl-adapter-parquet/tests/*
+            - src/adapter/etl-adapter-meilisearch/tests/*
+            - src/adapter/etl-adapter-logger/tests/*
+            - src/adapter/etl-adapter-json/tests/*
+            - src/adapter/etl-adapter-http/tests/*
+            -  src/adapter/etl-adapter-csv/tests/*
         - identifier: missingType.iterableValue
```

Comment:
fixing missingType with phpstan level 8 fixing this issue is pretty much pointless as it can be fixed by adding param array<mixed> which gives pretty much nothing so we would need to upgrade to level 9 (which I absolutely have no time to deal with, already invested waaaay to much into static analysis for arguable benefits) so if anyone is interested in doing that - feel free